commandapi-plugin-test/src/test/java/ArgumentTests.java

```diff
@@ -396,18 +389,18 @@ public void executionTestWithListArgument() {
 				type.set((List<String>) args[0]);
 			})
 			.register();
-	
-		server.dispatchCommand(sender, "listfunc cat, wolf, axolotl");
-		server.dispatchCommand(sender, "listfunc cat, wolf, axolotl, chicken");
-		server.dispatchCommand(sender, "listfunc axolotl, wolf, chicken, axolotl");
-		server.dispatchCommand(sender, "listfunc axolotl, wolf, chicken, axolotl, " + sender.getName());
-	
-		assertEquals(List.of("cat", "wolf", "axolotl"), type.get());
+
+		server.dispatchCommand(sender, "listfunc cat, wolf, axolotl"); // normal list
+		server.dispatchCommand(sender, "listfunc cat, wolf, axolotl, wolf"); // don't allow duplicates
+		server.dispatchCommand(sender, "listfunc axolotl, wolf, chicken, cat"); // don't allow unknown items
```

Comment:
In the tests, it's better to use `assertThrows` here. I'd rather have a valid `assertThrows` that actually throws rather than the `assertNull` at the bottom which could mean anything

Reply:
You can use the `server.dispatchThrowableCommand` method which wraps commands via Brigadier to let you test whether commands fail or not, for example:
```java
assertThrows(CommandSyntaxException.class, () -> assertTrue(server.dispatchThrowableCommand(sender, "listfunc cat, wolf, axolotl, wolf")));
```